src/Stratis.Bitcoin.Features.LightWallet/Blocks/ILightWalletBlockStoreService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.LightWallet.Blocks
+{
+    public interface ILightWalletBlockStoreService : IDisposable
+    {
+        ChainedHeader PrunedUpToHeaderTip { get; }
```

Comment:
Please add some details about what these do.

Reply:
OK I've added lots of comments on the interface etc. 👍